data/hai/hai reveal 2 start.txt

```diff
@@ -233,7 +233,7 @@ mission "Hai Rescue: Ooonem"
 			branch nohint
 				and
 					not "Hai Rescue: Ooonem Hint: offered"
-					not "Hai Rescue: Ooonem (defer count)" >= 1
+					"Hai Rescue: Ooonem (defer count)" < 1
```

Comment:
Couldn't this just be:
```suggestion
					not "Hai Rescue: Ooonem (defer count)"
```
or am I missing something?

Reply:
I think you're right, and your version looks nicer.